config/routes.rb

```diff
@@ -31,9 +35,9 @@
       get "/complete" => "subscriptions#complete", as: :subscription
     end
 
-    get "/authenticate" => "authentication#sign_in", as: :sign_in
-    post "/authenticate" => "authentication#request_sign_in_token", as: :request_sign_in_token
-    get "/authenticate/process" => "authentication#process_sign_in_token", as: :process_sign_in_token
+    # DEPRECATED: legacy routes for managing subscriptions
+    get "/authenticate" => "subscriber_authentication#sign_in"
+    get "/authenticate/process" => "subscriber_authentication#process_sign_in_token"
```

Comment:
It might be good to flag this with the date we can delete it from if this will soon be safe to remove.

Reply:
Updated - how does it look now?